README.md

```diff
@@ -56,19 +56,19 @@ Live Server
 ### Setup
 
 Clone this repository to your desired folder: 
-git@github.com:Gmanlove/Hello-Microverse.git
+https://github.com/tomdan-ai/Hello-Microverse.git
 
 
```

Comment:
- [ ] In your MIT.md file, kindly replace the placeholder `[YOUR NAME]` with your name and every occurrence of `[APP TYPE]` with the name of your application. This makes your LICENSE personalized and defined 😎 👍